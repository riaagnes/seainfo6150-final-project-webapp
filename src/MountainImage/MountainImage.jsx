import React from "react";
import styles from "../MountainImage/MountainImage.module.css";

const MountainImage = (props) => {
  return <img className={styles.image} alt={props.text} src={props.url} />;
};

export default MountainImage;
