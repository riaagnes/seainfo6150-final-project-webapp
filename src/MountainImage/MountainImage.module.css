.image {
    border: 10px solid rgb(215, 225, 228);
    border-radius: 50%;
    box-shadow: 0px 10px 10px 0 rgba(0, 0, 0, 0.5);
    margin-right: 20px;
    margin-bottom: 30px;
    width: 150px;
    height: 100px;
}