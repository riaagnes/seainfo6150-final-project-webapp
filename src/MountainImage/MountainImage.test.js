import React from "react";
import MountainImage from "./MountainImage";

describe("MountainImage tests", () => {
  const props = {
    url:
      "https://upload.wikimedia.org/wikipedia/commons/b/b4/Mt_si_and_meadowbrook_cows.jpg",
    title: "si mountain",
  };

  it("renders correctly", () => {
    const { container } = render(<MountainImage {...props} />);
    expect(container).toMatchSnapshot();
  });
});
