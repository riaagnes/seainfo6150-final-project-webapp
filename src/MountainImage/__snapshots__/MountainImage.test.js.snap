// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MountainImage tests renders correctly 1`] = `
<div>
  <img
    class="image"
    src="https://upload.wikimedia.org/wikipedia/commons/b/b4/Mt_si_and_meadowbrook_cows.jpg"
  />
</div>
`;
